how-to/virtualisation/qemu.md

```diff
@@ -35,6 +60,147 @@ qemu-system-x86_64 options image[s]
 
 So take a look at the [QEMU manpage](http://manpages.ubuntu.com/manpages/bionic/man1/qemu-system.1.html), [`qemu-img`](http://manpages.ubuntu.com/manpages/bionic/man1/qemu-img.1.html) and the [QEMU documentation](https://www.qemu.org/documentation/) and see which options best suit your needs.
 
-## Next steps
+While a standard QEMU configuration works for most use cases, some scenarios demand high-vCPU VMs. In the next section, we’ll cover how to create QEMU virtual machines with up to 1024 vCPUs.
```

Comment:
Love your ordering of the 2 additional pages here - this is a logical progression and the context you added makes this flow really nicely.